assets/CET_to_py.py

```diff
@@ -31,12 +34,25 @@
 
 but some have shorter, more convenient aliases, some of which are 
 inspired by Matplotlib colormaps of the same name and others
-based on the qualitative appearance.
+based on the qualitative appearance.  The colormaps with 
+shorter names tend to be the most useful subset, and for
+cases like automatic population of a GUI widget these
+colormaps are are provided as a separate subset:
+
+  palette['name'] or palette_n.name
+  cm_n['name'] or cm_n.name
 """
 
-__version__ = '0.9.0'
+__version__ = '0.9.1'
 
 from collections import OrderedDict
+
+class AttrODict(OrderedDict):
+    """Ordered dictionary with attribute access (e.g. for tab completion)"""
+    def __getattr__(self, name): return self[name]
+    def __delattr__(self, name): del self[name]
+    def __setattr__(self, name, value): self[name] = value
```

Comment:
Should this also have a ``__dir__`` method for tab completion or would that be too cluttered:

```python
    def __dir__(self): return self.keys()
```